editor/editor_node.cpp

```diff
@@ -7305,7 +7305,16 @@ EditorNode::EditorNode() {
 	}
 
 	remove_android_build_template = memnew(ConfirmationDialog);
-	remove_android_build_template->set_ok_button_text(TTR("Show in File Manager"));
+
+#if defined(WINDOWS_ENABLED)
```

Comment:
imho, this repeating logic + strings should be extracted into a separate centralized function.

Now it's repeated over 4 files, with exactly the same logic and strings
Could be just a simple helper-function.

Reply:
Not worth it, I think. Especially that the string should be coming ideally from the OS instead of being fed directly in the code.